### PR TITLE
faq: Add meaningful questions

### PR DESCRIPTION
**Pull request by sils**
Imported from the wiki.

### CODE REVIEW

**Comment by hemangsk** [comment]
ack 59572d5

File: data/faq.js [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
 {
-	"ques": "Question 2",
-	"ans" : "Answer"
-},
-{
-	"ques": "Question 3",
-	"ans" : "Answer"
+	"ques": "How do I work out a great proposal?",
+	"ans" : "Please design your project in a way that you have **multiple \
+checkpoints**. We'd like you to specify at least three partial goals so if \
```
Comment (sims1253) [inline data/faq.js]: Milestones/issues for checkpoints?

File: data/faq.js [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+something goes wrong for a valid reason, we can rearrange your schedule \
+meaningfully. Each of those goals has to be merged into the main coala \
+codebase in order for it to be considered \"complete\". You should be aware \
+of what that involves for you (writing tests, going through extensive code \
+review, learning :), reiterating).\
```
Comment (sims1253) [inline data/faq.js]: writing docs

File: data/faq.js [hunk 3]
```diff
@@ -1,14 +1,26 @@
...
+\
+You can use any project idea listed here or one that came out of your mind - \
+however be sure to make us aware of your plans and start planning your project \
+with us so we can coordinate and - in the worst case - prohibit waste of time \
+from your side. At all cost make sure you find something that interests you. \
+We want this to be a fun and educating experience for you!\
+\
+Use the \"draft\" feature on the GSoC system, submit your draft early, get a \
```
Comment (sims1253) [inline data/faq.js]: link to the draft thingy?

Comment (sils) [inline data/faq.js]: there is no link, it's in the gsoc system which isn't even online yet

**Pull request merged**